### PR TITLE
New custom 404 page

### PR DESCRIPTION
**Pull request by dmundra**
@garaynakca here is the draft pull request for the 404 page. The branch is called `new-404-page`.

----
:books: Documentation preview :books:: https://civicactions-handbook--1043.org.readthedocs.build/en/1043/

### CODE REVIEW

File: .config/theme_overrides/404.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+{% extends "main.html" %}
+
+<!-- Content -->
+{% block content %}
+<h1>We looked, but couldn’t quite find that page.</h1>
+<h3>
+  The page you are looking for has been removed, is temporarily unavailable or
+  does not exist. Maybe try the address again, or head to the homepage.
+</h3>
```
Comment (dmundra) [inline .config/theme_overrides/404.html]: Preview https://civicactions-handbook--1043.org.readthedocs.build/en/1043/not-found

I think having a header for all that text is not good practice for accessibility. Are you looking to make the text larger?

Comment (garaynakca) [reply .config/theme_overrides/404.html]: @dmundra Yes.. I was just hoping to bump up the text size.

Comment (garaynakca) [reply .config/theme_overrides/404.html]: .... but, it's okay if it has to be the standard body size.

Comment (dmundra) [reply .config/theme_overrides/404.html]: You can wrap that in a div and give it a unique class. Then CSS can be added to this file https://github.com/CivicActions/guidebook/blob/new-404-page/assets/css/extra.css to style it. Do you want to try that?

**Comment by dmundra** [comment]
Crossing-linking comments https://github.com/CivicActions/guidebook/commit/98c8dc1f1021f37d755b77d84beac113035b318f#comments

File: .config/theme_overrides/404.html (hunk 1 above)
Comment (garaynakca) [reply .config/theme_overrides/404.html]: Thanks for CSS guidance!  Will try that!

File: assets/css/extra.css [hunk 2]
```diff
@@ -23,6 +23,13 @@ font-size of other elements in the theme.
 .rst-versions .rst-other-versions {
   color: #a9a9a9 !important;
 }
+
+/* adding a larger type option for lead text
```
Comment (dmundra) [inline assets/css/extra.css]: @garaynakca looks like you missed adding the closing comment `*/` to this line. I think the CSS won't work till that is added.

**Review by garaynakca — APPROVED** [review]

**Comment by garaynakca** [comment]
@dmundra The preview for this branch displays the readme/homepage rather than the 404.  Is there a way to preview the 404?

**Comment by dmundra** [comment]
> @dmundra The preview for this branch displays the readme/homepage rather than the 404. Is there a way to preview the 404?

@garaynakca, ya try this URL https://civicactions-handbook--1043.org.readthedocs.build/en/1043/not-found (I just put random text at the end to try a page that doesn't exist).

File: .config/theme_overrides/404.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+  IN THE SOFTWARE.
+-->
+
+{% extends "main.html" %}
+
+<!-- Content -->
+<div class="leadtext">404</div>
```
Comment (dmundra) [inline .config/theme_overrides/404.html]: @garaynakca looks like we lost the block `{% block content %}` to go above this div.

Comment (dmundra) [reply .config/theme_overrides/404.html]: I believe that is causing the errors.

Comment (garaynakca) [reply .config/theme_overrides/404.html]: Okay... thanks!  Fixing that.

**Comment by garaynakca** [comment]
Hi, @dmundra .  Does the CSS also update with the commits?  I made a change to the leadtext class, but don't see it displaying here: https://civicactions-handbook--1043.org.readthedocs.build/en/1043/not-found     
Thanks!

**Comment by dmundra** [comment]
> Hi, @dmundra . Does the CSS also update with the commits? I made a change to the leadtext class, but don't see it displaying here: https://civicactions-handbook--1043.org.readthedocs.build/en/1043/not-found Thanks!

Hey @garaynakca, it does. You might need to do a cache clear. This is what I am seeing:

Screenshot of 404 page after CSS changes applied

Is that match what you are seeing?

**Comment by garaynakca** [comment]
@dmundra Got it!  Thank you...  starting to look closer.

**Comment by ContentBrewmaster** [comment]
@garaynakca Would like us to close out the 2 PRs related to the designs you created for the 404 and home page. Is there anything @dmundra or I can do to help you?

File: .config/theme_overrides/404.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+{% block content %}
+<div class="leadtext">404</div>
+<h1>We looked, but couldn’t quite find that page.</h1>
+<h2>
+  The page you are looking for has been removed, is temporarily unavailable or
+  does not exist. Maybe try the address again, or head to the
+  <a href="{{ config.extra.homepage | d(nav.homepage.url, true) | url }}">
+    homepage.
+  </a>
+</h2>
```
Comment (dmundra) [inline .config/theme_overrides/404.html]: @garaynakca I think using headers for this one is not best practice. I think '404' should be the h1 and the other two should be divs with styling you would like to see.

**Pull request merged**